ui/lesson/Terminal.tsx

```diff
@@ -95,7 +103,7 @@ export default function Terminal({ success, lines, next, onChange }) {
             className="react-terminal-wrapper react-terminal-light relative box-border w-full rounded text-lg "
             onClick={autoFocus}
           >
-            <div className="react-terminal flex h-[600px] flex-col overflow-auto">
+            <div className="react-terminal flex max-h-[65vh] flex-col overflow-auto md:max-h-[80vh]">
```

Comment:
The bottom status bar is still hidden(depending on the height of the mobile device), the best solution will be to make the terminal adjust according to the screen size but I have given that a try and it seems like a difficult implementation because the topbar and tabs offset the height a little bit. I think this implementation will work for the time being.

Reply:
Considering how tall the bottom bar is, it might be best to not have it fixed but simply part of the normal layout flow (on mobile)? Just throwing the idea out there.

![image](https://user-images.githubusercontent.com/695901/226541297-f691fda0-ea9b-445a-b5dd-75c14548270d.png)

Reply:
Is this height thing maybe a separate issue since the title talks about Regex?